connect.php

```diff
@@ -4,7 +4,7 @@
 $host = 'localhost';
 $db   = 'infinitoAdmin';
 $user = 'root';
-$pass = DBPASSWORD;
+$pass = '12345678';
```

Comment:
@StrixROX follow the instructions mentioned in updated [contributing.md](https://github.com/Infinito-IIT-Patna/Infinito2020/blob/master/CONTRIBUTING.md) to setup the database.

Reply:
Well sorry about that! Will resolve and send pics/vids asap

Reply:
Added the gallery link:
![image](https://user-images.githubusercontent.com/40167899/103159044-a6b91980-47ea-11eb-8ab9-4111a9a4034d.png)

Demo of gallery:
https://user-images.githubusercontent.com/40167899/103159179-17ad0100-47ec-11eb-8553-9d2a9f32a537.mp4